testing/kuttl/e2e/pgbouncer/11--open-connection.yaml

```diff
@@ -0,0 +1,39 @@
+---
+# Connect through PgBouncer and wait long enough for TLS certificates to rotate.
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: psql-open-connection
+  labels: { postgres-operator-test: kuttl }
+spec:
+  backoffLimit: 1
+  template:
+    metadata:
+      labels: { postgres-operator-test: kuttl }
+    spec:
+      restartPolicy: Never
+      volumes:
+        # TODO(cbandy): Provide a CA bundle that clients can use for verification.
+        - { name: tls, secret: { secretName: proxied-cluster-cert } }
+      containers:
+        - name: psql
+          image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.1-0
+          env:
+            # Connect through PgBouncer.
+            - name: PGURI
+              valueFrom: { secretKeyRef: { name: proxied-pguser-proxied, key: pgbouncer-uri } }
+
+            # Verify the certificate presented by PgBouncer.
+            - { name: PGSSLMODE, value: verify-full }
+            - { name: PGSSLROOTCERT, value: /mnt/ca.crt }
+
+          volumeMounts:
+            - { name: tls, mountPath: /mnt }
+
+          command:
+            - psql
+            - $(PGURI)
+            - -qAt
+            - --set=ON_ERROR_STOP=1
+            - --command=SELECT pid, backend_start FROM pg_stat_activity WHERE pid = pg_backend_pid();
+            - --command=SELECT pg_sleep_for('5 minutes');
```

Comment:
Yes. There's no assert that waits for this job. Can I do more to call that out?

Reply:
So the connection remains open, but the pod itself counts as a completed job?

Reply:
Yeah maybe worth another quick comment. Something like this?
```suggestion
            - --command=SELECT pg_sleep_for('5 minutes');
            # Wait 5 minutes so that open-connection job stays active while other test steps complete
```

Reply:
🤔 [`Job.status.active`](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/job-v1/#JobStatus) includes pending pods. Perhaps the assertion should try to find the _running_ pod. I'll make some updates.

Reply:
Done.